go.mod

```diff
@@ -1,54 +1,40 @@
 module code.cloudfoundry.org/app-autoscaler-cli-plugin
 
-go 1.19
+go 1.22
 
 require (
-	code.cloudfoundry.org/cli v7.1.0+incompatible
+	code.cloudfoundry.org/cli v0.0.0-20240404012325-79ae2fb5db92
```

Comment:
The latest is - according to <https://pkg.go.dev/code.cloudfoundry.org/cli> – v7.1.0+incompatible. However this is outdated information. The most recent one is from July. Can we please switch to that one?

Reply:
I will update to 7.7.11

Reply:
Upgraded to 7.7.11